package Auth.exception;

public class UserNotFoundException extends RuntimeException {
    public UserNotFoundException() {
    }
}
